diff --git machine-learning/immich_ml/models/__init__.py b/machine-learning/immich_ml/models/__init__.py
index 6c3a74c96..d52a0b8e0 100644
--- machine-learning/immich_ml/models/__init__.py
+++ machine-learning/immich_ml/models/__init__.py
@@ -3,8 +3,6 @@ from typing import Any
 from immich_ml.models.base import InferenceModel
 from immich_ml.models.clip.textual import MClipTextualEncoder, OpenClipTextualEncoder
 from immich_ml.models.clip.visual import OpenClipVisualEncoder
-from immich_ml.models.ocr.detection import TextDetector
-from immich_ml.models.ocr.recognition import TextRecognizer
 from immich_ml.schemas import ModelSource, ModelTask, ModelType
 
 from .constants import get_model_source
@@ -30,12 +28,6 @@ def get_model_class(model_name: str, model_type: ModelType, model_task: ModelTas
         case ModelSource.INSIGHTFACE, ModelType.RECOGNITION, ModelTask.FACIAL_RECOGNITION:
             return FaceRecognizer
 
-        case ModelSource.PADDLE, ModelType.DETECTION, ModelTask.OCR:
-            return TextDetector
-
-        case ModelSource.PADDLE, ModelType.RECOGNITION, ModelTask.OCR:
-            return TextRecognizer
-
         case _:
             raise ValueError(f"Unknown model combination: {source}, {model_type}, {model_task}")
 
