--- server/src/schema/migrations/1744910873969-InitialMigration.ts	2025-04-29 09:42:51
+++ server/src/schema/migrations/1744910873969-InitialMigration.ts.new	2025-04-29 09:42:45
@@ -29,7 +29,11 @@
   await sql`CREATE EXTENSION IF NOT EXISTS "cube";`.execute(db);
   await sql`CREATE EXTENSION IF NOT EXISTS "earthdistance";`.execute(db);
   await sql`CREATE EXTENSION IF NOT EXISTS "pg_trgm";`.execute(db);
-  await sql`CREATE EXTENSION IF NOT EXISTS "vectors";`.execute(db);
+  if (vectorExtension === DatabaseExtension.VECTORS) {
+    await sql`CREATE EXTENSION IF NOT EXISTS "vectors";`.execute(db);
+  } else {
+    await sql`CREATE EXTENSION IF NOT EXISTS "vector";`.execute(db);
+  }
   await sql`CREATE OR REPLACE FUNCTION immich_uuid_v7(p_timestamp timestamp with time zone default clock_timestamp())
   RETURNS uuid
   VOLATILE LANGUAGE SQL
